Mathlib/Topology/ContinuousFunction/Bounded.lean

```diff
@@ -1512,4 +1514,134 @@ lemma norm_sub_nonneg (f : α →ᵇ ℝ) :
 
 end
 
+section CompactlySupported
+
+variable (α γ : Type*) [TopologicalSpace α] [NormedRing γ]
+
+/-- The two-sided ideal of compactly supported functions. -/
+def compactlySupported : TwoSidedIdeal (α →ᵇ γ) :=
+  .mk' {z | HasCompactSupport z} .zero .add .neg' .mul_left .mul_right
```

Comment:
With slight modifications, this all works in non-unital rings, and you don't want `α γ` to be explicit everywhere
```suggestion
/-- The two-sided ideal of compactly supported functions. -/
def compactlySupported (α γ : Type*) [TopologicalSpace α] [NonUnitalNormedRing γ] :
    TwoSidedIdeal (α →ᵇ γ) :=
  .mk' {z | HasCompactSupport z} .zero .add .neg' .mul_left .mul_right
  
variable {α γ : Type*} [TopologicalSpace α] [NonUnitalNormedRing γ]
```